### ISSUE TITLE
getting an error when deploying the operator

### ISSUE THREAD

**Issue by raffaelespazzoli** [jetstack/cert-manager-olm#8]
I'm seeing this error hen trying to deploy the cert-manager operator after the creation of the cert-manager CR:
```
Failed to pull image "registry.connect.redhat.com/jetstack/cert-manager-controller:v0.15.0-alpha.2-ubi": rpc error: code = Unknown desc = Error reading manifest v0.15.0-alpha.2-ubi in registry.connect.redhat.com/jetstack/cert-manager-controller: manifest unknown: manifest unknown
```

**Comment by raffaelespazzoli** [comment]
the issue is the that image the operator is trying to pull has a `-ubi` prefix which does not exists in the  repo.

**Comment by meyskens** [comment]
This should be fixed in v0.15 (the stable release). It should be released to the OperatorHub any moment now by the build system. Could you try that one out?
Thanks a lot for the feedback!
